types/node-forge/index.d.ts

```diff
@@ -95,10 +96,40 @@ declare module "node-forge" {
     }
 
     namespace ssh {
+        interface FingerprintOptions {
+            /**
+             * @description the delimiter to use between bytes for `hex` encoded output
+             */
+            delimiter?: string;
+            /**
+             * @description if not specified, the function will return `ByteStringBuffer`
+             */
+            encoding?: 'hex'|'binary';
```

Comment:
To remain consistent with the rest of the file, put spaces around `|` characters.